packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4997,6 +5001,8 @@ class Semantics extends SingleChildRenderObjectWidget {
   ///
   /// This setting is often used in combination with [SemanticsConfiguration.isSemanticBoundary]
   /// to create semantic boundaries that are either writable or not for children.
+  /// 
+  /// Setting [properties.route] to true will force this value to be true as well.
```

Comment:
This kind of magic is usually a bad sign... Can you talk more about this? Why is it necessary?

If we really need it then we should have a separate constructor for this case.

Reply:
Removed - I don't think this is necessary since most users will only interact with the route semantics through routeName.  Otherwise the correct way to handle this is probably in merging logic.